server/src/core/routes/app.routes.ts

```
@@ -1,4 +1,21 @@
+  sinChewDaily: (await import('@lib/sinChewDaily/server')).default,
+  railwayMap: (await import('@lib/railwayMap/server')).default,
+  music: (await import('@lib/music/server')).default,
+  calendar: (await import('@lib/calendar/server')).default,
+  booksLibrary: (await import('@lib/booksLibrary/server')).default,
+  sudoku: (await import('@lib/sudoku/server')).default,
+  changiAirportFlightStatus: (await import('@lib/changiAirportFlightStatus/server')).default,
+  movies: (await import('@lib/movies/server')).default,
+  momentVault: (await import('@lib/momentVault/server')).default,
+  wallet: (await import('@lib/wallet/server')).default,
+  todoList: (await import('@lib/todoList/server')).default
+});
```

ESLint

[error] Expected blank line before this statement.

(padding-line-between-statements)